### DISCUSSION TITLE
getting CORS when accessing dashboard

### DISCUSSION THREAD

**Discussion by bilalbayasut** [IceWhaleTech/CasaOS#1994] [Q&A]
Has anyone ever faced problem where apps doesn't shows up, and when I check the network log, it said 
```
strict-origin-when-cross-origin
```
on  
http://192.168.1.6/v2/app_management/web/appgrid

seems getting appgrid endpoint took too long time

**Comment by CDU-Ge** [comment]
> Has anyone ever faced problem where apps doesn't shows up, and when I check the network log, it said 
> ```
> strict-origin-when-cross-origin
> ```
> on  
> http://192.168.1.6/v2/app_management/web/appgrid
> 
> seems getting appgrid endpoint took too long time

I encountered the same issue as well. It seemed to be due to authentication issues, but I'm not sure about the details. 
Did you manage to resolve it?
